configs/default/DYST-DYSF44530D.config

```diff
@@ -0,0 +1,133 @@
+board_name DYSF44530D
```

Comment:
Please read and follow the instructions on https://github.com/betaflight/unified-targets (**including testing**) - this was clearly not done by following the instructions, as it is missing the first line.